tests/surveyPropertyMapsTestUtils.py

```diff
@@ -180,7 +180,7 @@ def makeMockVisitSummary(visit,
     return visit_summary
 
 
-class MockVisitSummaryReference(lsst.daf.butler.DeferredDatasetHandle):
+class MockVisitSummaryReference(lsst.pipe.base.InMemoryDatasetHandle):
```

Comment:
I'm not sure I understand why these are not using the `inMemoryDataset` attribute as was done in `assembleCoaddTestUtils.py`.

Reply:
I've fixed on #758 -- there was a bit of subtlety in the tests reusing the ref since InMemoryDatasetHandle is a frozen data class.